<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QMainWindow" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>520</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Go!</string>
    </property>
    <property name="icon">
     <iconset resource="../res/backy.qrc">
      <normaloff>:/Backy/document-save.png</normaloff>:/Backy/document-save.png</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupSource">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>&amp;Source drive or folder</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>30</y>
       <width>561</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="layoutSource">
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="sourceFolder"/>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="labelSourceDescription">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Browse for the source folder or drive you want to backup the contents of.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="browseSource">
        <property name="text">
         <string>Browse...</string>
        </property>
        <property name="icon">
         <iconset resource="../res/backy.qrc">
          <normaloff>:/Backy/folder-open.png</normaloff>:/Backy/folder-open.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupTarget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>601</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>&amp;Backup location</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>561</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="layoutTarget">
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="labelTargetDescription">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Browse for the backup destination folder or drive.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="browseTarget">
        <property name="text">
         <string>Browse...</string>
        </property>
        <property name="icon">
         <iconset resource="../res/backy.qrc">
          <normaloff>:/Backy/folder-open.png</normaloff>:/Backy/folder-open.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="targetFolder"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>601</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Backup &amp;progress</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>561</width>
       <height>164</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="layoutStatus">
      <item>
       <widget class="QLabel" name="labelStatusCurrentDescription">
        <property name="text">
         <string>Current item</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelStatusCurrentName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="labelStatusCurrentProgress">
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="textDirection">
         <enum>QProgressBar::TopToBottom</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelStatusOverallDescription">
        <property name="text">
         <string>Overall progress</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="labelStatusOverallProgress">
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="labelStatusAbort">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>210</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Abort</string>
     </property>
     <property name="icon">
      <iconset resource="../res/backy.qrc">
       <normaloff>:/Backy/process-stop.png</normaloff>:/Backy/process-stop.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionOnlineDocumentation"/>
    <addaction name="actionAboutBacky"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../res/backy.qrc">
     <normaloff>:/Backy/system-log-out.png</normaloff>:/Backy/system-log-out.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOnlineDocumentation">
   <property name="icon">
    <iconset resource="../res/backy.qrc">
     <normaloff>:/Backy/internet-web-browser.png</normaloff>:/Backy/internet-web-browser.png</iconset>
   </property>
   <property name="text">
    <string>Online &amp;documentation</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAboutBacky">
   <property name="icon">
    <iconset resource="../res/backy.qrc">
     <normaloff>:/Backy/help-browser.png</normaloff>:/Backy/help-browser.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About Backy</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../res/backy.qrc">
     <normaloff>:/Backy/preferences-system.png</normaloff>:/Backy/preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/backy.qrc"/>
 </resources>
 <connections/>
</ui>
